Sources/AltSwiftUI/Source/Views/Controls/SecureField.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  SecureField.swift
+//  AltSwiftUI
+//
+//  Created by Elvis Lin on 2020/12/1.
+//
+
+import UIKit
+
+/// A view that allows text input in one line.
+public struct SecureField: View {
```

Comment:
Currently, our TextField view can be setup to show secure entry. Did you consider wrapping the TextField view instead and configure it to show secure entry in the SecureField's body? That will reduce duplication between the 2 components and allow for easier scaling.

Reply:
Thanks for your suggestion, and the pull request has been updated.